assets/dev/js/editor/elements/views/base.js

```diff
@@ -621,11 +633,7 @@ BaseElementView = BaseContainer.extend( {
 		}
 	},
 
-	renderOnChange( settings ) {
-		if ( ! this.allowRender ) {
-			return;
-		}
-
+	applyChanges( settings, render = true ) {
```

Comment:
Rename to indicate that the return value is a bool - whether to render to not

Reply:
Handled.